src/Auth0Client.ts

```diff
@@ -1308,4 +1317,16 @@ export default class Auth0Client {
       return entry.access_token;
     }
   }
+
+  /**
+   * Releases any lock acquired by the current page that's not released yet
+   *
+   * Get's called on the `pagehide` event.
+   * https://developer.mozilla.org/en-US/docs/Web/API/Window/pagehide_event
+   */
+  private _releaseLockOnPageHide = async () => {
```

Comment:
Is there a reason to define this way rather than the way used throughout the class already?

Reply:
When calling `addEventListener` and `removeEventListener`, I believe we want to ensure we pass the exact same callback instance. We can't use a method, because using a method like this: 

```
addEventListener('...', this.someMethod)
```
Would lose track of the correct `this`, and calling `bind()` would make it a different instance.
Making it an arrow ensures we can just use `this.someFunctionDefinedAsAField` and still use `this` in side of it while ensuring we are always passing the exact same function around.

Reply:
Ahh I had overlooked that, thanks for clarifying!